api/src/gameState/gameState.controller.ts

```diff
@@ -162,4 +198,21 @@ export class GameStateController {
     this.gatewayService.emitGameState(newGameState);
     return newGameState;
   }
+
+  @Delete(':id')
+  async deleteGame(@Param('id', ParseIntPipe) id: number) {
+    if (!id) {
+      throw new HttpException('Missing parameter', 400);
+    }
+    return this.gameStateService.deleteGame({ id });
```

Comment:
Whenever the game does not exist, the answer of this function must be true IMO

Reply:
Agree